Source/Tooltips/Hooks.lua

```diff
@@ -174,8 +174,12 @@ hooksecurefunc (GameTooltip, "SetInboxItem",
 hooksecurefunc("InboxFrameItem_OnEnter",
   function(self)
     local itemCount = select(8, GetInboxHeaderInfo(self.index))
-    local tooltipEnabled = AUCTIONATOR_SHOW_MAILBOX_TIPS == 1 and  (
-      AUCTIONATOR_V_TIPS == 1 or AUCTIONATOR_A_TIPS == 1 or AUCTIONATOR_D_TIPS == 1
+    local O = Auctionator.Config.Option
```

Comment:
This confused the hell out of me lol. Can we either use the fully qualified name, or a complete word for the temporary variable (i.e. `ConfigOptions`). Also, since this doesn't change, I think we should declare it outside of the function as a local to this module for a minor memory save.